src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,39 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, $width);
+  gap: 48px;
+
+  // display: flex;
+  // flex-wrap: wrap;
+
+  // max-width: 944px;
+  margin: 0 auto;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 200px;
+    height: 300px;
+    border: 1px solid #ccc;
```

don't forget to create variables for all colors

```suggestion
    border: 1px solid $c-color;
```